core/src/components/router-link/router-link.tsx

```diff
@@ -36,6 +36,13 @@ export class RouterLink implements ComponentInterface {
    */
   @Prop() routerDirection: RouterDirection = 'forward';
 
+  /**
+   * Specifies where to display the linked URL.
+   * Only applies when an `href` is provided.
+   * Special keywords: `"_blank"`, `"_self"`, `"_parent"`, `"_top"`.
+   */
+  @Prop() target: string | undefined;
```

Comment:
wondering why not using `?`?
```suggestion
  @Prop() target?: string;
```

Reply:
Because this is how we have it done for all other components using `AnchorInterface` and if we did decide to add an interface for this it would already be working. https://github.com/ionic-team/ionic/pull/17883/files#diff-36f66a20486cf32eda71ad087e5998bcR55

Reply:
Gotcha!